packages/react/src/components/Alert/Alert.tsx

```diff
@@ -153,17 +177,44 @@ class Alert extends AutoControlledComponent<WithAsProp<AlertProps>, AlertState>
   }
 
   renderContent = ({ styles, accessibility }: RenderResultConfig<AlertProps>) => {
-    const { actions, dismissible, dismissAction, content } = this.props
+    const { actions, dismissible, dismissAction, content, icon, header, body } = this.props
 
     return (
       <>
-        {Box.create(content, {
+        {Icon.create(icon, {
           defaultProps: {
-            className: Alert.slotClassNames.content,
-            styles: styles.content,
-            ...accessibility.attributes.content,
+            className: Alert.slotClassNames.icon,
+            styles: styles.icon,
+            xSpacing: 'after',
```

Comment:
We can move this to styles instead?

Reply:
But `xSpacing` is a prop of `Icon`. If I set it in styles prop, the value is not taken into account.

Reply:
### Problem for styles

It's just a `margin` value. We should avoid styling via props as it created hidden dependencies. We have already discussed this problem and it's related to children API.

```tsx
<Alert icon="book" />
// This things look equal 🤔 
// But rendered result will be different as there are no matching styles for it
<Alert>
  <Icon name="book" />  
</Alert>
// With usage component props we are making it even more complicated
```

### To coupled

If `horizontalSpace` variable in `IconVariables` will be changed, it will accidentally affect this component. Will be it valid?

Reply:
Fixed